recipes/libsecret/all/conanfile.py

```diff
@@ -14,8 +15,8 @@ class LibsecretConan(ConanFile):
     generators = "pkg_config"
 
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
+    options = {"shared": [True, False], "fPIC": [True, False], "gcrypt": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "gcrypt": True}
```

Comment:
Please reformat + use `with_libgcrypt`.
That's the format we often use to let the user know that the option will add support for an extra package.
```suggestion
    options = {
        "shared": [True, False],
        "fPIC": [True, False],
        "with_libgcrypt": [True, False],
    }
    default_options = {
        "shared": False,
        "fPIC": True,
        "with_libgcrypt": True,
    }
```